std/stdio.d

```diff
@@ -4683,18 +4683,19 @@ alias stdin = makeGlobal!(StdFileHandle.stdin);
 @safe unittest
 {
     // Read stdin, sort lines, write to stdout
-    import std.algorithm.mutation : copy;
     import std.algorithm.sorting : sort;
     import std.array : array;
+    import std.range.primitives : put;
     import std.typecons : Yes;
 
     void main() {
-        stdin                       // read from stdin
-        .byLineCopy(Yes.keepTerminator) // copying each line
-        .array()                    // convert to array of lines
-        .sort()                     // sort the lines
-        .copy(                      // copy output of .sort to an OutputRange
-            stdout.lockingTextWriter()); // the OutputRange
+        put(
```

Comment:
Case in-point: the only reason this worked is because `stdout.lockingTextWriter` represents an output range which has only one global output destination.

Reply:
Boo, this breaks the UFCS chain. Is there as UFCS-able `put` in Phobos?